meta-lmp-base/recipes-security/optee/optee-os-mfgtool_git.bb

```diff
@@ -1,3 +1,7 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/optee-os:"
 
 require optee-os_${PV}.bb
+
+EXTRA_OEMAKE_append_imx = " \
```

Comment:
Can you add this change to meta-lmp-bsp/recipes-security/optee/optee-os-mfgtool_%.bbappend instead?

That way the base recipe keeps without any bsp specific information, for users consuming our layer.